Library/Homebrew/formula.rb

```diff
@@ -2219,7 +2219,17 @@ def method_added(method)
     # Shows when running `brew info`.
     #
     # <pre>license "BSD-2-Clause"</pre>
-    attr_rw :license
+    def license args=nil
+      if args.blank?
+        return @licenses
```

Comment:
```suggestion
      if args.nil?
        return @licenses
```

A rare time to use `nil?`!